linux_os/guide/system/software/sudo/var_sudo_logfile.var

```diff
@@ -0,0 +1,16 @@
+documentation_complete: true
+
+title: 'Sudo - logfile value'
+
+description: |-
+    Specify the sudo logfile to use. The default value used here matches the example
+    location from CIS, which uses /var/log/sudo.log.
+
+interactive: false
```

Comment:
Setting this to true will enable SCAP Workbench to edit/add values during tailoring. In case one doesn't want to use suggested value from CIS.

Reply:
Good catch! Yeah it's definitely useful for the user to be able to override this location and put the logfile where they want. I've pushed a new commit that sets this to `true` now. :+1: